recipe/meta.yaml

```diff
@@ -10,16 +17,24 @@ source:
   sha256: 65e086708ae56670676b7b2340ad82b901d8c9993d1241a839c8990bdb8d6212
   patches:
       - patches/0000-Consolidated-build-fixes-for-XLA.patch  # [not win]
+      - patches/0001-Add-missing-typename.patch
 
 build:
-  number: 0
+  number: {{ build }}
   # s390x is missing bazel.
   skip: true  # [s390x or py<310]
+  skip: true  # [(cuda_compiler_version != "None") and aarch64]
+  skip: true  # [(cuda_compiler_version != "None") and win]
```

Comment:
I don't think these two lines are necessary - cuda_compiler_version is always "None" on non-linux-64 platforms